scripts/hostcfgd

```diff
@@ -1302,84 +1302,141 @@ class NtpCfg(object):
     1) ntp-config.service handles the configuration updates and then starts ntp.service
     2) Both of them start after all the feature services start
     3) Purpose of this daemon is to propagate runtime config changes in
-       NTP, NTP_SERVER and LOOPBACK_INTERFACE
+       NTP, NTP_SERVER, NTP_KEY, and LOOPBACK_INTERFACE
     """
+    NTP_CONF_RESTART = ['service', 'ntp-config', 'restart']
```

Comment:
Why is service being used instead of systemctl?

Reply:
No special reason. Both will work equally.

Reply:
Regardless, I think systemctl should be used instead of service here, since the systemctl command is already used in many other places, and this would keep things consistent.